src/App.js

```diff
@@ -4,6 +4,8 @@ import './App.css'
 import Editor from './views/Editor'
 import VowelPlotter from './VowelPlotter'
 
+import {list} from './VowelList'
```

Comment:
The exports here changed a bit 🙈 

You can use `import vowels from './VowelList'` now, or whatever name you'd like for the default export.